network/server.go

```diff
@@ -324,8 +324,10 @@ func (s *Server) setupBootnodes() error {
 // checkPeerCount will attempt to make new connections if the active peer count is lesser than the specified limit.
 func (s *Server) checkPeerConnections() {
 	for {
+		delay := time.NewTimer(10 * time.Second)
```

Comment:
I would recommend using one timer and `Reset()` for less alloc.

Reply:
085bf03 replace it